app/javascript/styles/mastodon/admin.scss

```diff
@@ -248,6 +248,11 @@ $content-width: 840px;
         display: inline-flex;
         flex-flow: wrap;
         gap: 5px;
+        align-items: center;
```

Comment:
This ensures a heading actions that contains both text and a button doesn't look maligned.